src/Fantomas.Tests/LambdaTests.fs

```diff
@@ -561,12 +548,9 @@ Target.create "Install" (fun _ ->
     |> should
         equal
         """
-Target.create
-    "Install"
-    (fun _ ->
-        Yarn.install (fun o -> { o with WorkingDirectory = clientDir })
-        // Paket restore will already happen when the build.fsx dependencies are restored
-        )
+Target.create "Install" (fun _ -> Yarn.install (fun o -> { o with WorkingDirectory = clientDir })
```

Comment:
This one looks odd

Reply:
Feels like it should not be considered single-line expression because of the following comment

Reply:
This is unrelated to the changes in this PR, created #1864 to follow up.